packages/crayons-core/src/components/data-table/custom-cells/user/custom-cell-user.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'fw-custom-cell-user',
+  styleUrl: 'custom-cell-user.scss',
+  shadow: true,
+})
+export class CustomCellUser {
+  @Prop() image = null;
+
+  @Prop() name = '';
+
+  @Prop() email = '';
+
+  getInitialsFromName() {
+    const initials = this.name
+      ? this.name
+          .split(' ')
+          .map((name) => name[0].toUpperCase())
+          .join('')
+      : '';
+    return initials;
+  }
+
+  render() {
+    return (
+      <div class='name-box-container'>
+        <div class='avatar'>
+          <fw-avatar
+            size='small'
+            initials={this.getInitialsFromName()}
+            image={this.image}
+          ></fw-avatar>
+        </div>
+        <div class='name-box'>
+          <div class='name-box-text'>{this.name}</div>
```

Comment:
Would there be any truncation happening to content if the column size is smaller?
We need to review the details with the Design team.

Reply:
As we discussed, currently column expands to the length of the content currently. But this logic will be changed. We will be adding 'min' and 'max-width' to columns. We will allow users to set width to a column by passing width as one of the properties of the column.